mlflow/openai/utils.py

```diff
@@ -99,3 +101,51 @@ def request(*args, **kwargs):
             return original(*args, **kwargs)
 
     return _mock_request(new=request)
+
+
+class _OAITokenHolder:
+    def __init__(self, api_type):
+        self.api_type = api_type
+        self._api_token = None
```

Comment:
Shouldn't this default to `openai.api_key`?

Reply:
`_api_token` contains a token (complex object). `openai.api_key` contains the token secret (string).